packages/sit-onyx/src/types/components.ts

```diff
@@ -46,3 +46,14 @@ export type BaseSelectOption<TValue extends SelectOptionValue = SelectOptionValu
 };
 
 export type SelectOptionValue = string | number | boolean;
+
+export type AutofocusProp = {
```

Comment:
set it on TextArea + select too?

Reply:
maybe a question for our dev lead @JoCa96  :)

Reply:
Done